### PR TITLE
Javascript refactoring and packages updates

### PR DESCRIPTION
**Pull request by marioradu05**

### CODE REVIEW

File: src/Admin/templates/admin/list.html.twig [hunk 1]
```diff
@@ -72,17 +69,14 @@
...
-                    <h4 class="modal-title" id="deleteFormModalTitle">Delete Admin</h4>
+                    <h4 class="modal-title" id="deleteFormModalTitle">Confirm Delete</h4>
                     <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close">
                     </button>
                 </div>
-                <div class="modal-body">
-                    <div id="deleteFormMessages"></div>
-                    <div id="deleteFormPlaceholder"></div>
-                </div>
+                <div class="modal-body"></div>
```
Comment (alexmerlin) [inline src/Admin/templates/admin/list.html.twig]: This way the `Message::ADMIN_DELETED_SUCCESSFULLY` message does not get displayed.

**Review by alexmerlin — CHANGES_REQUESTED** [review]

**Review by alexmerlin — CHANGES_REQUESTED** [review]
- `src/App/assets/js/components/*.js` - most of the files have no/multiple empty lines at the end of the file
- also, [this error](https://insight.symfony.com/projects/6a7ecfc1-a0ed-4901-96ac-d0ff61f7b55f/analyses/180) reported by SymfonyInsight

File: public/images/app/logo-dark.svg
Comment (pinclau) [inline public/images/app/logo-dark.svg:1]: this file should not exist

File: public/images/app/logo-light.svg
Comment (pinclau) [inline public/images/app/logo-light.svg:1]: this file should not exist

**Review by pinclau — DISMISSED** [review]

File: public/images/app/logo-light.svg
Comment (marioradu05) [reply public/images/app/logo-light.svg:1]: Removed unused logo.

File: public/images/app/logo-dark.svg
Comment (marioradu05) [reply public/images/app/logo-dark.svg:1]: Removed unused logo.

**Review by alexmerlin — APPROVED** [review]

**Review by arhimede — APPROVED** [review]

**Pull request merged**